### PR TITLE
Bump hook to 0.182.

### PR DESCRIPTION
**Pull request by cjwagner**
/cc @BenTheElder 
ref #5584

### CODE REVIEW

File: prow/cluster/deck_deployment.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
         args:
         - --build-cluster=/etc/cluster/cluster
         - --tide-url=http://tide/
-        - --hook-url=http://hook:8888/plugin-help
+        - --hook-url=http://hook:0.182/plugin-help
```
Comment (cjwagner) [inline prow/cluster/deck_deployment.yaml]: Boo

File: prow/cluster/starter.yaml [hunk 2]
```diff
@@ -184,7 +184,7 @@ spec:
       - name: deck
         image: gcr.io/k8s-prow/deck:0.66
         args:
-        - --hook-url=http://hook:8888/plugin-help
+        - --hook-url=http://hook:0.182/plugin-help
```
Comment (BenTheElder) [inline prow/cluster/starter.yaml]: also here :smile:

**Comment by krzyzacy** [comment]
maybe it's time to switch to date-sha tag :-)

**Comment by BenTheElder** [comment]
date-sha and bump all the things is the way to go, deploying them with image rules and content hashes is even better though :-)

**Comment by cjwagner** [comment]
I just fixed the sed regex for now, but I agree that we should switch to date-sha tags.

**Pull request merged**